programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,7 @@
+function solution(seoul) {
+    
+    for (let i=0; i<seoul.length ;i++) {
+        if (seoul[i] ==='Kim')
+             return (`김서방은 ${i}에 있다`);
+        }
+        }
```

Comment:
```suggestion
    for (let i = 0; i < seoul.length; i++) {
        if (seoul[i] ==='Kim') return (`김서방은 ${i}에 있다`);
        }
    }
```
공백 컨벤션, 들여쓰기 신경써서 작성해주세요